crates/sui-types/src/gas.rs

```diff
@@ -292,9 +292,13 @@ impl<'a> SuiGasStatus<'a> {
         Ok(())
     }
 
-    pub fn charge_min_tx_gas(&mut self) -> Result<(), ExecutionError> {
+    /// Try to charge the minimal amount of gas from the gas object.
+    /// This function is called in tx signing phase to make sure
+    /// the gas object has enough balance to cover minimal transaction cost.
```

Comment:
Is that true? Shouldn't that be checked during signing?

Reply:
yeah this function is only called during signing.